src/Symfony/Component/Security/Http/Tests/EventListener/IsCsrfTokenValidAttributeListenerTest.php

```diff
@@ -130,6 +163,27 @@ public function testIsCsrfTokenValidCalledCorrectlyWithInvalidTokenKey()
         $listener->onKernelControllerArguments($event);
     }
 
+    public function testExceptionWhenInvalidExpressionId()
+    {
+        $this->expectException(\RuntimeException::class);
```

Comment:
I did not see where you throw this RuntimeException ?

Reply:
It's thrown when the argument is not found. maybe the the test case should be removed?

Reply:
I'd either drop it, or document it and laybe test the mesage content.


By just reading the test here it was really not clear what the problem was and how i could have fix it

Reply:
are you OK that I remove it for now? :)

Reply:
removed in a fixup commit until removal is approved

Reply:
Oh i have no "ok" to give here :) 

I'm **_trying_** to help where i can, but i have not this kind of decision to make :)) 

Personnaly i would probably remove it.. .but then maybe create a specific test to check the InvalidCsrf...Exception is thrown with the correect message when it should ?

Reply:
Me I'd remove it too, was added to show a failing test case for not found argument;
But thinking about it twice it may not be the responsibility of the new feature, as the token id will always end up being an string.
And if the argument is not found, it's probably already tested under the ExpressionLanguage component scope